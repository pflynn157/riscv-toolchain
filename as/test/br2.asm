    beq x0, x0, LBL2
LBL1:
    addi x1, x2, 20
    addi x2, x3, 20
LBL2:
    beq x0, x0, LBL1
