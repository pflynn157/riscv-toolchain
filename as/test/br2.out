00000663
01410093
01418113
fe000ce3
